lib/fs.js

```diff
@@ -132,7 +132,7 @@ function makeCallback(cb) {
   }
 
   return function() {
-    return cb.apply(null, arguments);
+    return cb(...arguments);
```

Comment:
The current next-tick benchmarks do not test the code modified here. I'll add a commit to include this code path in the benchmarks.